nest/00. Foreword.md

```diff
@@ -0,0 +1,24 @@
+# Getting started with Nest
+
+## Foreword
+
+Welcome to the official guide to get you started with Nest! This guide assumes you are using the latest version of Nest, which is 0.1.0 at the time of writing.
+
+In order to avoid any confusion, when we refer to `Nest` (with an upper case n), we refer to the project as a whole. When we refer to `nest` (with a lower case n), we only refer to the binary (executable file) nest. Thus, `finest` refer to its eponymous binary, and is included in Nest.
+
+[//]: # (TODO: add link to the installation instructions page of Raven-OS)
+Nest is a dependency-based package manager for [Raven-OS](https://raven-os.org), which is an adaptive, customizable and evolutive open-source Operating System (OS) based on the Linux kernel. To use Nest, it's recommended to have installed Raven-OS to run Nest on it, because Nest may overlap with an already installed package manager. To install Raven-OS, follow the instructions [here](). Nest is included in the installation of Raven-OS by default, so you can start using it right away.
```

Comment:
there are five sentences in what you highlighted. Did you meant to say 'paragraph' instead of 'sentence'?

Reply:
"To use Nest, it's recommended to install Raven-OS and to run Nest in it, because Nest may overlap with an already installed package manager."
Doesn't seem so bad now that I wrote it out.